<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSimConnectSettingsComponent</class>
 <widget class="QWidget" name="CSimConnectSettingsComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>159</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="vl_SettingsComponent">
   <item>
    <widget class="QGroupBox" name="gb_FsxSimConnectConfigFile">
     <property name="title">
      <string>FSX/P3D SimConnect config file</string>
     </property>
     <layout class="QFormLayout" name="fl_Settings">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_SettingsFsxAddress">
        <property name="text">
         <string>Address</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="le_SettingsFsxAddress">
        <property name="text">
         <string>127.0.0.1</string>
        </property>
        <property name="maxLength">
         <number>128</number>
        </property>
        <property name="placeholderText">
         <string>e.g. &quot;127.0.0.1&quot; or &quot;192.128.3.1&quot;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbl_SettingsFsxPort">
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="le_SettingsFsxPort">
        <property name="text">
         <string>500</string>
        </property>
        <property name="placeholderText">
         <string>normally &quot;500&quot;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lbl_SettingsFsxExistsSimconncetCfg">
        <property name="toolTip">
         <string>is 'SimConnect.cfg' available?</string>
        </property>
        <property name="text">
         <string>.cfg?</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QWidget" name="wi_SettingsFsxExistsSimconncetCfg" native="true">
        <layout class="QHBoxLayout" name="hl_SettingsFsxExistsSimconncetCfg">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="le_SettingsFsxExistsSimconncetCfg">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string>local &quot;SimConnect.cfg&quot; file?</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_SettingsFsxExistsSimconncetCfg">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>check</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <layout class="QHBoxLayout" name="hl_SettingsSimconnectCfgButtons">
        <property name="spacing">
         <number>3</number>
        </property>
        <item>
         <widget class="QPushButton" name="pb_SettingsFsxOpenSimconnectCfg">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>open</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_SettingsFsxDeleteSimconnectCfg">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>del.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_SettingsFsxSaveSimconnectCfg">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_SettingsFsxTestConnection">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Test connection</string>
          </property>
          <property name="text">
           <string extracomment="Test connection">test</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
